examples/fmm/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+file(
+        GLOB_RECURSE
+        MY_SRC
+        "*.cpp"
+        "*.h"
+        "../../tests/testAutopas/testingHelpers/RandomGenerator.cpp"
+)
+
+add_executable(fmm ${MY_SRC})
+
+target_link_libraries(fmm autopas)
+
+# add the test to checkExamples
+add_dependencies(checkExamples fmm)
```

Comment:
you might actually want to add a test, which might also be a run of the fmm executable.

You might orient yourself on the other tests, e.g.,
The test will only pass if 0 is returned from COMMAND, so you might also use grep or sth. else to add some more functionality.
```
# -----------------test-----------------
# add check for current target
# cmake-format: off
add_test(
    NAME fmm.test
    COMMAND fmm
    CONFIGURATIONS checkExamples
)
# cmake-format: on
```